extensions/wikia/AdEngine/js/slot/service/megaAdUnitBuilder.js

```diff
@@ -45,10 +46,9 @@ define('ext.wikia.adEngine.slot.service.megaAdUnitBuilder', [
 			],
 			'PX': ['INVISIBLE_SKIN', 'INVISIBLE_HIGH_IMPACT', 'INVISIBLE_HIGH_IMPACT_2'],
 			'HiVi': ['INCONTENT_BOXAD_1', 'MOBILE_IN_CONTENT'],
-			'VIDEO': ['FEATURED', 'OUTSTREAM', 'UAP_BFAA', 'UAP_BFAB', 'ABCD', 'VIDEO']
+			'VIDEO': ['FEATURED', 'INCONTENT_PLAYER', 'OUTSTREAM', 'UAP_BFAA', 'UAP_BFAB', 'ABCD', 'VIDEO']
```

Comment:
Are you sure that we want to load display slot (incontent_player) as wka1a.VIDEO group? All others relates to VAST/video

Reply:
Ok, I'll leave it as OTHER then.